code/src/components/Footer.js

```diff
@@ -0,0 +1,93 @@
+import React from "react";
+import styled from "styled-components/macro";
+
+const Container = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  background-color:  #adebeb;
+  border-top: 2px solid white;
+  width: 100%;
+  height: 80px;
+  margin: 0;
+  position: fixed;
+  top: 86%;

```

Comment:
remove this` top: 86%` and add` bottom: 0` Right now there's a little gap in the bottom so when you scroll a large image or the actual scanner you can see it below/behind the footer.